gitlint/utils.py

```diff
@@ -21,6 +21,72 @@
 import pathlib2 as pathlib
 
 
+# copy from python3, shutil.which
```

Comment:
Which version of python was used? Which commit?

Probably is better to depend on a backported version of which like. https://pypi.org/project/backports.shutil_which/

Reply:
Thanks for your patience, I will improve this PR later。